media-gfx/exiv2/files/exiv2-0.25-install-dirs.patch

```diff
@@ -0,0 +1,22 @@
+--- a/config/CMakeChecks.txt	2015-07-19 14:17:22.857046105 +0200
```

Comment:
A patch should basically have the same format as a commit message at the top, also see https://dev.gentoo.org/~vapier/clean-patches